plugins/colorbutton/plugin.js

```diff
@@ -154,6 +154,23 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					attributes: { role: 'listbox', 'aria-label': lang.panelTitle }
 				},
 
+				select: function( callback ) {
```

Comment:
API docs for the new method are missing.

Reply:
I'm not sure about namespace of the docs :thinking: Probably `CKEDITOR.ui.colorButton` would be best but I'm not a fan of introducing empty namespace for this single private method. WDYT about just private docs i.e. commented out with `//` (so commenting private functions according to https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#210-comments)?

Reply:
Sounds good 👍